src/cookbook/design/themes.md

```diff
@@ -53,89 +86,113 @@ MaterialApp(
 );
 ```
 
-See the [`ThemeData`][] documentation to see all of
-the colors and fonts you can define.
+Most instances of `ThemeData` set values for the following two properties. These properties affect the entire app.
 
-## Themes for part of an application
+1. [`colorScheme`][] defines the colors.
+1. [`textTheme`][] defines text styling.
 
-To override the app-wide theme in part of an application,
-wrap a section of the app in a `Theme` widget.
+[`colorScheme`]: {{site.api}}/flutter/material/ThemeData/colorScheme.html
+[`textTheme`]: {{site.api}}/flutter/material/ThemeData/textTheme.html
 
-There are two ways to approach this: creating a unique `ThemeData`,
-or extending the parent theme.
+To learn what colors and fonts you can define,
+check out the [`ThemeData`][] documentation.
 
-{{site.alert.note}}
-  To learn more, watch this short Widget of the Week video on the Theme widget:
+## Apply a theme
 
-  <iframe class="full-width" src="{{site.youtube-site}}/embed/oTvQDJOBXmM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-{{site.alert.end}}
+To apply your new theme, add the `Theme.of(context)` method within a widget's styling properties. These include, but are not limited to, `style` and `color`.
```

Comment:
- Semantic breaks
- It's not just the `Theme.of` call, so perhaps "use" instead of "add"?
- Not all widgets include those two properties, so we should clarify that.

```suggestion
To apply your new theme, use the `Theme.of(context)` method 
when specifying a widget's styling properties.
These can include, but are not limited to, `style` and `color`.
```